documentation/docs/instructor/assessment.mdx

```diff
@@ -0,0 +1,207 @@
+
+<Callout variant={CalloutVariant.tip}>
+    <p>Using actual student submissions as training examples makes the training more realistic and reduces instructor workload.</p>
+</Callout>
+
+## Double-Blind Assessment
+
+Manual assessment in Artemis is **double-blind**:
+
+- Reviewers don't see student names during assessment
+- Students don't see reviewer identities
+
+This minimizes bias and increases assessment objectivity.
+
+
+## Handling Complaints
+
+After receiving an assessment, students can submit **complaints** if they believe the assessment needs revision _(if you've enabled this feature)_.
+
+### Complaint Token System
+
+- Instructors set a maximum number of complaints per student per course
+- Each complaint uses one "token"
+- If accepted, the token is returned to the student
+- Students can submit unlimited complaints as long as their complaints are accepted
+
+### Complaint Review Process
+
+1. Student submits a complaint with justification
+2. Reviewer accesses the complaint from the Exercise Dashboard
+3. Reviewer re-evaluates the submission and decides:
+
+| Decision   | Action & Screenshot                                                                                                                                                                                     |
+|------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Accept** | Add or modify feedback blocks (positive or negative points) and click "Accept complaint"<br/><Image src={acceptComplaintImg} alt={"Accept Complaint"} size={ImageSize.large} hideBorder={true}></Image> |
+| **Reject** | Explain why the complaint is rejected and click "Reject complaint"<br/><Image src={rejectComplaintImg} alt={"Reject Complaint"} size={ImageSize.large} hideBorder={true}></Image>                       |
+| **Unsure** | Remove the lock so another reviewer can evaluate<br/><Image src={lockImg} alt={"Unlock Complaint"} size={ImageSize.large} hideBorder={true}></Image>                                                    |
+
+<Callout variant={CalloutVariant.warning}>
+    <p>Complaints can result in score increases or decreases based on the re-assessment. Be thorough and fair.</p>
+</Callout>
+
+
+## More Feedback Requests
+
+Students can also submit **More Feedback Requests** to ask for clarification without disputing their score.
+
+<Callout variant={CalloutVariant.info}>
+    <p>More feedback requests don't cost tokens, but reviewers <strong>cannot change the score</strong> when responding.</p>
+</Callout>
+
+The review process is identical to handling complaints for reviewers. Although, there are [differences for students](/student/assessment#more-feedback-requests).
```

Comment:
```suggestion
Students can also submit **More Feedback Requests** to ask for clarification without disputing their score.
The review process is identical to handling complaints for reviewers.
Although, there are [differences for students](/student/assessment#more-feedback-requests).

<Callout variant={CalloutVariant.info}>
    <p>More feedback requests don't cost tokens, but reviewers <strong>cannot change the score</strong> when responding.</p>
</Callout>
```
I would put the important information together, then afterwards put the callout with the additional stuff.